### DISCUSSION TITLE
Some details on BOM parts please!

### DISCUSSION THREAD

**Discussion by EclipseBoom** [open-plc-com/PLC-modem#3] [Q&A]
Hello,

Thanks for this project!  This is the most simple and clean PLC modem design I have seen.

I'm very interested in this for communication with a water pump driver down in a well.  Only the two DC power wires are available so this is a perfect add-on with minimal parts count and cost.  Cable length is about 50m.

Can you please give some details on the transformer.  Is this a standard 1:1:1 communication transformer?  Does it have any particular inductance value or volt-time values required?

Also the ferrite.  I assume that is part F1. You mentioned in the description wanting a 1.8MHz frequency.  What is the impedence value needed for that?

Thanks again.

Chris

**Comment by open-plc-com** [comment]
Hi Chris,

transformer:
Core: B64290P0037X049
3 coils with 4 turns each

Finished product: you can buy it on order here: www.xfullstartech.com 

Regards,
Alexandr

**Comment by EclipseBoom** [comment]
Thank you for the reply.  That web site does not exist that I can find.

Do you know the specification for that winding pattern with that core?  Inductance and so forth.  With only 4 turns it would seem to be a low inductance.  Perhaps these specs do not matter so much.  I want to buy a packaged 1:1:1 transformer not do winding myself so I can make many units of this.

Thanks,

Chris

**Comment by open-plc-com** [comment]
Try this: https://www.xfullstartechs.com
Trans. coil:
Characteristic resistance about 320 Ohms
Inductance about 28 uHn
F = 1.84 MHz (amateur frequency)

**Comment by EclipseBoom** [comment]
What do you think about this part: https://www.digikey.com/en/products/detail/w%C3%BCrth-elektronik/744253200/6564967
